### DISCUSSION TITLE
Pynest Community support.

### DISCUSSION THREAD

**Discussion by Ismail-Mirza** [PythonNest/PyNest#70] [Q&A]
Is it good idea to go industry grade project with Pynest as it is new and a lot to build ?

**Comment by ItayTheDar** [comment]
> Is it good idea to go industry grade project with Pynest as it is new and a lot to build ?

Hi @Ismail-Mirza 
I'm the creator of PyNest, and i'm using it at my company - Lemonade. I'm using pynest as the framework for my mivro service and i can say that it worked very good. In terms of scalability and performance it is the same as fastapi, and in terms of dev experience, managing dependencies across our platform and the time it takes us to ship new features to production it is really change my team's life. We are also now aligned with the rest of our company micro services (writing with NestJS).

If you have any doubts or question, i'm here to help 😃

**Comment by Ismail-Mirza** [comment]
Thank you for your reply. But the pynest server is not auto-reloading. I
have to restart the server every time I make a change in code.

**Reply by ItayTheDar** [reply]
Since the most common server to run with is uvicorn, you can just add a flad that said reload=True and you are tood to go

**Comment by Ismail-Mirza** [comment]
I did so but still not auto reloading. Thank you.

**Comment by 8syncdev** [comment]
How to set up security in Pynest, I dont see document about middleware, hash, token related in this topic
